src/guiFormSpecMenu.h

```diff
@@ -252,18 +261,18 @@ class GUIFormSpecMenu : public GUIModalMenu
 		}
 		StaticTextSpec(const std::wstring &a_text,
 				const core::rect<s32> &a_rect):
-			text(a_text),
 			rect(a_rect),
 			parent_button(NULL)
 		{
+			text = unescape_string(remove_escapes(a_text));
```

Comment:
We have two escape systems now, have you thought about giving it a better name that makes it easier to distinguish the two systems?